aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java

```diff
@@ -39,7 +39,7 @@
 import com.dylibso.chicory.runtime.OpcodeImpl;
 import com.dylibso.chicory.runtime.StackFrame;
 import com.dylibso.chicory.runtime.exceptions.WASMRuntimeException;
-import com.dylibso.chicory.wasm.Module;
```

Comment:
Changed to avoid the overlap with `java.lang.Module`

Reply:
I don't think we need to rename, as `java.lang.Module` is not typically used in application code. We use `com.google.inject.Module` everywhere and the name overlap doesn't cause problems.

That said, I'm not against the rename if we think it's better. Let's do it as a separate commit or PR.

Reply:
ok, I revert the change in this PR and we keep the conversation going.